### PR TITLE
Fix Mod Configs

### PR DESCRIPTION
**Pull request by kemus**
Hey, big fan of your setup tool. It's the only thing that's let me come back to the BG series for a long long time.

I was playing around with parsing your mod configs in the ee-mod-setup (trying to see if I can eventually make a sort of mod manager in Python instead of autoit), and as part of it, I noticed that some mod configs were hard to parse. 

I think these cases are errors in the .ini files, and that fixing them should help ee-mod-setup parse them as well.

Please let me know if you'd like me to make any changes, or if I completely misunderstood things and these files are actually correct.

Thanks!

### CODE REVIEW

File: App/Config/Global/Candlemem.ini [hunk 1]
```diff
@@ -16,7 +16,7 @@ Tra=0
 @0=Candlekeep Memories
 Tra=1
 
-[WeiDU-RU]
+[WeiDU-GE]
```
Comment (kemus) [inline App/Config/Global/Candlemem.ini:19]: I believe this should be GE, we have a RU up above and [Mod]'s Tra= contains GE:2.

File: App/Config/Global/Charlatan.ini [hunk 2]
```diff
@@ -6,7 +6,7 @@ Link=https://github.com/Raduziel/Charlatan-Kit/blob/master/Charlatan/Charlatan.i
 Down=https://github.com/Raduziel/Charlatan-Kit/archive/master.zip
 Save=Charlatan-Kit-master.zip
 Size=921600
-Tra=EN:0;FR:1,RU:2
+Tra=EN:0,FR:1,RU:2
```
Comment (kemus) [inline App/Config/Global/Charlatan.ini:9]: These are comma separated everywhere but here

File: App/Config/Global/LongerRoadEE.ini [hunk 3]
```diff
@@ -28,9 +28,9 @@ Tra=2
 @1=Przywr�cenie oryginalnego portretu Irenicusa z SoA.
 Tra=3
 
-[WeiDU-EN]
+[WeiDU-RU]
```
Comment (kemus) [inline App/Config/Global/LongerRoadEE.ini:31]: I think this should be a RU, [Mod]'s Tra= has RU:4

File: App/Config/Global/LongerRoadEE.ini [hunk 4]
```diff
@@ -28,9 +28,9 @@ Tra=2
...
 @0=Longer Road 
-@1=Restore Irenicus's original portrait from SoA.
+@1=Vosstanovite original'nyy portret Irenikusa iz SoA.
```
Comment (kemus) [inline App/Config/Global/LongerRoadEE.ini:33]: I don't speak russian, this is just a google translate

File: App/Config/Global/TomeAndBlood.ini [hunk 5]
```diff
@@ -1,6 +1,6 @@
 [Mod]
 Name=Tome and Blood
-Version=GitHub Latest Release
+Rev=GitHub Latest Release
```
Comment (kemus) [inline App/Config/Global/TomeAndBlood.ini:3]: Called 'Rev=' everywhere else

File: App/Config/Global/divine_remix.ini [hunk 6]
```diff
@@ -6,7 +6,7 @@ Link=https://www.gibberlings3.net/mods/kits/cleric/
 Down=https://github.com/Gibberlings3/Divine_Remix/archive/master.zip
 Save=Divine_Remix-master.zip
 Size=1417683
-Tra=EN:0,FR:1
+Tra=EN:0,FR:1,GE:2
```
Comment (kemus) [inline App/Config/Global/divine_remix.ini:9]: We have a [WeiDU-GR] but it doesn't show up here

File: App/Config/Global/eetact2.ini [hunk 7]
```diff
@@ -55,8 +55,8 @@ Tra=EN:0,IT:1,GE:2,FR:3,SP:4,PO:5,CH:6,RU:7
 Tra=0
 
 [WeiDU-IT]
-@100=Core Component
-@110=Ilyich Migliorato (richiede ToB)
```
Comment (kemus) [inline App/Config/Global/eetact2.ini:59]: These numbers don't match with [Weidu-EN]

File: App/Config/Global/eetact2.ini [hunk 8]
```diff
@@ -99,6 +99,7 @@ Tra=0
 Tra=1
 
 [WeiDU-GE]
+@0=Core Component
```
Comment (kemus) [inline App/Config/Global/eetact2.ini:102]: Missing core component

File: App/Config/Global/refinements.ini [hunk 9]
```diff
@@ -6,7 +6,7 @@ Link=https://github.com/UnearthedArcana/refinements
 Down=https://github.com/UnearthedArcana/refinements/archive/4.28.zip
 Save=refinements-4.28.zip
 Size=10233821
-Tra=EN:0,FR:1,JP:2,PO:3,RU:4
+Tra=EN:0,FR:1,JA:2,PO:3,RU:4
```
Comment (kemus) [inline App/Config/Global/refinements.ini:9]: Not sure which of "JA" or "JP" is correct for Japanese. I looked up the list of 2-letter language codes, and it seems to suggest I should use "JA", but I'm not sure if that's what you use elsewhere in your code.

File: App/Config/Global/viconia.ini [hunk 10]
```diff
@@ -6,7 +6,7 @@ Link=https://github.com/SpellholdStudios/ViconiaFriendship
 Down=https://github.com/SpellholdStudios/ViconiaFriendship/archive/v4.4.zip
 Save=ViconiaFriendship-4.4.zip
 Size=1854280
-Tra=EN:0,FR:1,RU:2,FR:3
```
Comment (kemus) [inline App/Config/Global/viconia.ini:9]: Removed duplicate language

File: App/Config/Global/whitequeen.ini [hunk 11]
```diff
@@ -36,7 +36,7 @@ Tra=3
 @0=Couleurs de l'Infini - La Reine Blanche
 @0?1_1=Installer l'ajout a la carte du monde d'origine (celle de BG2)
 @0?1_2=Installer l'ajout a la carte du monde pour BP-BGT
-Tra=0
+Tra=4
```
Comment (kemus) [inline App/Config/Global/whitequeen.ini:39]: [Mod]'s Tra= has this as 4

**Comment by kemus** [comment]
Added some comments to explain my reasoning -- I'm not used to contributing to open source, so I want to make sure I get the etiquette right. :)

File: App/Config/Global/Candlemem.ini (hunk 1 above)
Comment (RoxanneSHS) [reply App/Config/Global/Candlemem.ini:19]: Thanks, a typo.

File: App/Config/Global/Charlatan.ini (hunk 2 above)
Comment (RoxanneSHS) [reply App/Config/Global/Charlatan.ini:9]: Good catch!

File: App/Config/Global/LongerRoadEE.ini (hunk 3 above)
Comment (RoxanneSHS) [reply App/Config/Global/LongerRoadEE.ini:31]: Yes.

File: App/Config/Global/LongerRoadEE.ini (hunk 4 above)
Comment (RoxanneSHS) [reply App/Config/Global/LongerRoadEE.ini:33]: Not important, they do not show up anyway and tp2 reads from the mod's own tra files.

File: App/Config/Global/TomeAndBlood.ini (hunk 5 above)
Comment (RoxanneSHS) [reply App/Config/Global/TomeAndBlood.ini:3]: Typo

File: App/Config/Global/divine_remix.ini (hunk 6 above)
Comment (RoxanneSHS) [reply App/Config/Global/divine_remix.ini:9]: It is the other way round, I checked the mod and it is only EN, FR - other languages are commented out for EE update.

File: App/Config/Global/eetact2.ini (hunk 7 above)
Comment (RoxanneSHS) [reply App/Config/Global/eetact2.ini:59]: repaired

File: App/Config/Global/eetact2.ini (hunk 8 above)
Comment (RoxanneSHS) [reply App/Config/Global/eetact2.ini:102]: same

File: App/Config/Global/viconia.ini (hunk 10 above)
Comment (RoxanneSHS) [reply App/Config/Global/viconia.ini:9]: Checked in the mod, the second FR should be CH

File: App/Config/Global/whitequeen.ini (hunk 11 above)
Comment (RoxanneSHS) [reply App/Config/Global/whitequeen.ini:39]: corrected

**Comment by RoxanneSHS** [comment]
Great findings.
I have made the appropriate corrections.
Thanks a lot.
